src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1164,13 +1164,11 @@ def dbapi(
                 if field.name.lower() == column.lower():
                     column_type = field.datatype
             if column_type is None:
-                logger.error("Specified column does not exist in schema")
                 raise ValueError("Column does not exist")
 
             if not isinstance(column_type, _NumericType) and not isinstance(
                 column_type, DateType
             ):
-                logger.error(f"Unsupported column type: {column_type}")
                 raise ValueError(
                     f"unsupported type {column_type}, we support numeric type and date type"
```

Comment:
I suggest you make it more clear what supports numeric type and date type. Something like `unsupported type for {column_type} for ...`